.github/workflows/nightly_tests.yml

```diff
@@ -248,7 +248,7 @@ jobs:
           BIG_GPU_MEMORY: 40
         run: |
           python -m pytest -n 1 --max-worker-restart=0 --dist=loadfile \
-            -m "big_gpu_with_torch_cuda" \
+            -m "big_accelerator" \
```

Comment:
Should we not update the instances where the marker is actually present then? Example:
https://github.com/huggingface/diffusers/blob/7bc0a07b1947bcbe0e84bbe9ecf8ae2d234382c4/tests/lora/test_lora_layers_flux.py#L815C2-L815C25

I guess we should remove them?